install_config/topics/applying_configuration_changes.adoc

```diff
@@ -18,6 +18,17 @@ configuration changes, see xref:../install_config/master_node_configuration#mast
 # systemctl restart atomic-openshift-node
 ----
 
+[NOTE]
+====
+Kubernetes architecture expects reliable endpoints from cloud providers. When a
+cloud provider is down, the kubelet prevents {product-title} from restarting.
+If the underlying cloud provider endpoints are not reliable, do not install a
+cluster using cloud provider integration. Install the cluster as if it was in
+a no-cloud environment. It is not recommended to toggle cloud provider
```

Comment:
s/no-cloud/on-premise

Reply:
Thank you Kathryn. Would you be open to s/on-premise/bare metal/? My view is that the OpenStack provider could be on-premise and so that terminology may not clarify the point being made in this case.

Reply:
If that's the right nuance that you're trying to capture, yes.